tests/test_grid3d/test_grid_property_grdecl.py

```diff
@@ -48,69 +29,6 @@ def test_gridprop_to_from_file_is_identity(tmp_path, grid, data):
     assert_allclose(prop.get_npvalues1d(), prop_from_file.get_npvalues1d(), atol=1e-3)
 
 
-@pytest.mark.parametrize(
```

Comment:
Can you split the moving of these tests to a separate commit?

Reply:
Done!